README.md

```diff
@@ -19,13 +19,263 @@ The [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) spec is deliberately not dup
+            var link = document.createElement('link');
+            link.rel = 'stylesheet';
+            link.href = 'https://fonts.googleapis.com/css2?family=Open+Sans:wght@400&display=swap';
+            document.head.appendChild(link);
+          };
+      </script>
+    </head>
+  ```
+
+  **Optimized font subsetting using CSS:**
+  ```css
+  @font-face {
+    font-family: 'MyFont';
+    src: url('MyFontSubset.woff2') format('woff2'); /* Subset font file */
+    unicode-range: U+20-7E; /* Range for basic ASCII */
+  }
+
+  /* The full MyFont.woff2 will only be downloaded if characters outside of the basic ASCII range are used. */
+  @font-face {
+    font-family: 'MyFont';
+    src: url('MyFont.woff2') format('woff2'); /* Full font file */
+    unicode-range: U+80-FFFF; /* Rest of the unicode range */
+  }
+  ```
+  </blockquote>
+  </details>
+
+-
+  <details>
+    <summary>Use Old Style Figures for Inline Numbers for better legibility.
+    </summary>  
+    <blockquote>
+    Old style figures, or lowercase numbers, have varying heights and alignments, much like lowercase letters. When you have numbers inline with text, like in a paragraph, old style figures can blend in more naturally and disrupt the flow of text less than lining figures, or uppercase numbers.
+    </blockquote>
+  </details>
+
+-
+  <details>
+    <summary>Use Proportional Figures in Tables and Lists.
+    </summary>  
+    <blockquote>
+    Unlike old style figures which have varying heights and are suitable for running text, proportional figures all have the same height, which makes them line up better in tables and lists.
+    </blockquote>
+  </details>
+
+-
+  <details>
+    <summary>Letter Spacing should be adjusted proportional to Font Size.
+    </summary>  
+    <blockquote>
+
+    Just as line height needs to increase with font size to maintain readability, letter-spacing needs to decrease with font size. Larger font sizes naturally have more space between letters, and adding too much letter-spacing can disrupt the reading flow. 
+
+    Most font designers optimise spacing of their fonts for the font-size 16px/pt. Leave letter-spacing here at default 0. Small-Caps or All-Caps text should generally have more leter-spacing.
+
+    On dark backgrounds (darkmode), white characters appear tighter. To increase legibility increase letter-spacing.
+
+    When adjusting letter-spacing should always also adjust line-height, as the vertical whitespace influences the look of your horizontal whitespace.
+
+    [Read more about letter spacing in web interfaces here](https://www.webdesignerdepot.com/2020/07/the-designers-guide-to-letter-spacing/).
+    </blockquote>
+  </details>
+
+-
+  <details>
+    <summary>
+    Use Hanging Quotes: When using blockquotes or pull quotes, try to position quotation marks outside the text block. This is known as hanging quotes, and it can improve the overall visual alignment and rhythm of your text blocks.
+    </summary>
```

Comment:
```suggestion
    <summary>Use Hanging Quotes: When using blockquotes or pull quotes, try to position quotation marks outside the text block. This is known as hanging quotes, and it can improve the overall visual alignment and rhythm of your text blocks.</summary>
```